lib/utils/request.js

```diff
@@ -533,7 +448,7 @@ function getHeaders() {
   return headers;
 }
 
-function getProxy(url) {
+function getProxy() {
```

Comment:
Remove ignored param, also from callers due to linting. Might make it harder to reverse if the passed param is to be used again.